### DISCUSSION TITLE
Does the API have access to the 5-minute resolution data?

### DISCUSSION THREAD

**Discussion by melyux** [weather-gov/api#318] [Q&A]
**Please describe what you're trying to do.**
If I go to a station's page like [here](https://www.wrh.noaa.gov/mesowest/getobext.php?sid=KSGR&num=24&hfmetars=1), I can see data with 5-minute resolution. However, the API only returns the hourly data (at the :53 minute marks). Is there a way to access the more granular data? I haven't been able to find anything about the temporal resolution in the API documentation.

All the best.

**Comment by KTibow** [comment]
I just clicked on the "XML" page and got this: https://www.wrh.noaa.gov/mesowest/getobextXml.php?sid=KSGR&num=24

**Comment by melyux** [comment]
What is that? It’s loading a blank page for me.

**Comment by KTibow** [comment]
@melyux try curl:
```bash
curl "https://www.wrh.noaa.gov/mesowest/getobextXml.php?sid=KSGR&num=24"
```
I went to your page and clicked on the orange XML button at the top.

**Comment by paulnadler** [comment]
There was also a Json version of that API but it stopped working recently and would include QPF values which was nice.
https://www.wrh.noaa.gov/mesowest/getobextJson.php?sid=KSGR&num=24

In my experience using that API, the resolution isn't consistent. I think it depends on how often the weather station updates. I also wonder if this is an example of an API that will be replaced by the new API here.

**Comment by melyux** [comment]
Thanks for the XML link, I hadn't seen that. I guess I'm wondering why the `/stations/{stationId}/observations/latest` endpoint grabs the :53 minute mark data instead of the latest 5-minute granularity observation possible, as seen in the XML.

**Comment by StephenClouse** [comment]
API receives its data from [MADIS](https://madis.ncep.noaa.gov/).

At this time we do not receive the high-frequency METAR data.

**Comment by scadergit** [comment]
@paulnadler the short answer is that api.weather.gov will eventually replace the pseudo-apis written across the NWS. The regions were able to quickly develop some great ideas, but regional technology farms are getting phased out with a single approach at NCO.
